.github/workflows/compliance.yml

```diff
@@ -0,0 +1,49 @@
+name: Run compliance tests
+
+on:
+  workflow_dispatch:
+    inputs:
+#      test_repo_url:
+#        description: Tests repository (eg. owner/repo)
+#        required: true
+#        default: elixir-cloud-aai/tes-compliance-suite
+      server:
+        description: The API server URL
+        required: true
+      version:
+        description: The API version
+        required: true
```

Comment:
We should check _all_ tests, i.e., all versions, meaning that we wouldn't need a specific version param as well (or at least, the default for the test runner's validation mode - see above - should be to validate _all_ tests if no version is supplied)